src/main/java/com/ldtteam/structurize/client/gui/WindowScan.java

```diff
@@ -314,66 +312,50 @@ private void updateResources()
         Network.getNetwork().sendToServer(new UpdateScanToolMessage(pos1, pos2));
         
         final World world = Minecraft.getInstance().world;
-        resources.clear();
-        entities.clear();
+        resources = new HashMap<>(300);
```

Comment:
why new instead of clear?

Reply:
testing at midnight forgots cleaning next day